packages/useTrie/tsconfig.json

```diff
@@ -7,9 +7,7 @@
     "noLib": false,
     "jsx": "react",
     "sourceMap": true,
-    "downlevelIteration": true,
-    "types": ["jest", "node", "react", "react-dom"]
+    "downlevelIteration": true
   },
-  "typeRoots": ["./types"],
```

Comment:
This is not necessary. By default, TypeScript will look in your node modules for all `@types/*` packages and include them.

Reply:
This is me being irresponsible & not knowing what each option does... 

(I am sorry I resolved conversations thinking it was required to merge the PR...)

Reply:
No need to be sorry. 😉

Reply:
:wink: